src/main.py

```diff
@@ -380,8 +400,32 @@ def metric_page():
             ),
         ]
     )
+
     fig6.update_layout(yaxis_range=[0, MAE_LIMIT_DEFAULT])
-    st.plotly_chart(fig6, theme="streamlit")
+    
+    fig7 = go.Figure(
+        layout=go.Layout(
+        title=go.layout.Title(text="Daily Latest MAE All GSPs"),
+        xaxis=go.layout.XAxis(title=go.layout.xaxis.Title(text="Date")),
+        yaxis=go.layout.YAxis(title=go.layout.yaxis.Title(text="Error Value (MW)")),
+        legend=go.layout.Legend(title=go.layout.legend.Title(text="Chart Legend")),
+        )
+    )
+
+    fig7.add_traces(
+        go.Scatter(
+            x=df_mae_all_gsp["datetime_utc"],
+            y=df_mae_all_gsp["MAE All GSPs"],
+            mode="lines",
+            name="Daily Latest MAE All GSPs",
+            line=dict(color=line_color[4]),
+        ),
+        )
+    
+    if model_name in ["pvnet_v2", "cnn"]:
+        
+        st.plotly_chart(fig6, theme="streamlit")
```

Comment:
how come fig6 is also in this?

Reply:
so fig6 shows RMSE and there isn't any RMSE data when the `National_xg` model is selected. it just plots MAE, which is already displayed at top of the page. should I go ahead and leave the chart in?

Reply:
I think there shuold be a RMSE for `National_xg`, perhaps just it in for the moment, and we can sort it out if RMSE is not showing for `National_xg`